charmcraft.yaml

```diff
@@ -64,6 +70,13 @@ parts:
       rustup default stable
       craftctl default
 
+config:
+  options:
+    trusted-certificate-bundle:
+      type: string
+      description: >-
+        PEM certificate bundle to offer over the certificate_transfer relation.
```

Comment:
Should we mention "ca certs" specifically in the name of the config option or its description?

Reply:
I do not think it could confuse things further. It is true that most of the times, this will be used for CA certificates. However, it could also be used for leaf certificates also, as you can trust a certificate that is not a CA directly.